### PR TITLE
EX-179

### PR DESCRIPTION
**Pull request by paupujolasparset**

### CODE REVIEW

**Review by christiangda — APPROVED** [review]

**Review by llorenc-aizon — APPROVED** [review]

File: get-latest-matching-tag/action.yaml [hunk 1]
```diff
@@ -8,11 +8,9 @@ inputs:
   repository:
     description: 'Repository name'
     default: ${{ github.event.repository.name }}
-    required: true
   owner:
     description: 'Owner of the repository'
     default: ${{ github.repository_owner }}
-    required: true
   prefix:
     description: 'Prefix to match the tag against'
```
Comment (llorenc-aizon) [inline get-latest-matching-tag/action.yaml:15]: I guess if prefix not given, it just gets latest tag

File: jira-issues-query/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as core from '@actions/core';
-import queryHelper from '../src/helpers/jiraApi/queryHelper.js';
+import queryHelper from '../src/helpers/jiraApi/jqlSearch.js';
```
Comment (llorenc-aizon) [inline jira-issues-query/index.js:2]: Are you sure, in this case, **query** and **fields** are not **required**?

**Review by llorenc-aizon — APPROVED** [review]

File: get-latest-matching-tag/action.yaml (hunk 1 above)
Comment (paupujolasparset) [reply get-latest-matching-tag/action.yaml:15]: Yes, that's right

File: create-branches-from-tags/action.yaml
Comment (llorenc-aizon) [inline create-branches-from-tags/action.yaml:1]: As mentioned in https://github.com/aizonexecute/release-management/pull/1/files#r1297110839
_**Currently**, platform repository branches are created from branches
either **main** or another **release/x.x** branch_ - **not** from specific **tags**

Then, branch creation may fail for some repositories... 
one reason being branch already created in a previous failed run, 
or repository is archived (removed from main version but still in a patch)

**Pull request merged**